Main.qml

```diff
@@ -157,6 +153,8 @@ ApplicationWindow {
                     }
                 ]
                 state: "MINIMAL"
+                // textfield does not gain correct height on start, setting/unsetting text 'fixes' it
+                Component.onCompleted: { appmodel.resourceText = "_"; appmodel.resourceText = "" }
```

Comment:
@pieter-edelman-nictiz do you know how to fix?

Reply:
I think the problem is that the TextArea has no implicit height when it's empty, we were just lucky that it choose more or less the correct height before, when no placeholder was set.
So the logical fix would be to anchor the top and bottom to the `loadResourceButton`, which does have an implicit height. Just add:
```
anchors.top: loadResourceButton.top
anchors.bottom: loadResourceButton.bottom
```
somewhere in the InstanceEditor declaration and you should be fine.

Reply:
Yeah, worked... but strange that it didn't want to ask the textarea for the default height of a single line.

Feedback applied.